core/dive.c

```diff
@@ -4234,10 +4234,22 @@ static void delete_divecomputer(struct dive *d, int num)
 	invalidate_dive_cache(d);
 }
 
-/* always acts on the current dive */
-void delete_current_divecomputer(void)
+/* Clone a dive and delete goven dive computer */
```

Comment:
the commit message has a cut and paste error: 
https://github.com/Subsurface-divelog/subsurface/pull/2100/commits/872539e6974244c064b3d62e4561e0e6cba3995a says `Undo: make "delete dive computer front" undoable` and that `front` in there seems wrong.

Reply:
Fixed.